stl/src/xmath.hpp

```diff
@@ -104,11 +104,11 @@ union _Dval { // pun floating type as integer array
 unsigned short* _Pmsw(double*);
 
 short _Dint(double*, short);
-short _Dnorm(_Dval*);
-short _Dscale(double*, long);
+short _Dnorm(_Dval*) noexcept;
+short _Dscale(double*, long) noexcept;
 short _Dunscale(short*, double*);

```

Comment:
It's weird to mark some, but not all, of this internal machinery as `noexcept`. We should just get it all.